lib/helpers/rule-test-harness.js

```diff
@@ -365,7 +365,7 @@ module.exports = function generateRuleTests({
             delete expectedResults[i].rule;
             delete element.source;
 
-            assert.ok(element.message.includes(expectedResults[i].message));
+            assert.ok(element.message.includes(expectedResults[i].message), element.message);
```

Comment:
I don't think the message alone is enough, instead let's actually tell the user what we are asserting: `expected "foo" to contain "bar"`.